CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Changed
     
   Reported by @cded from @Bitovi
 
+* Escaped text in notifications
```

Comment:
Would be nice to include the reference to the Github PR number:
```suggestion
* Escaped text in notifications. #990
```

can we also move it from the "Changed" to the "Fixed" section below?